examples/confluent-example/pyproject.toml

```diff
@@ -6,12 +6,9 @@ authors = ["Marcos Schroh <marcos.schroh@kpn.com>"]
 
 [tool.poetry.dependencies]
 python = "^3.8"
-kstreams = "^0.3.0"
+kstreams = { path = "../../.", develop = true }
```

Comment:
I was doubting about it. On one hand is nice because it will use the current version of `kstreams` and if we have integration test we can always check whether the examples are broken, on other hand if someone copies the folder just to run the example it won't work. 

Maybe we should mention this in the `README.md` ?

Reply:
FMPOV users are expected to clone the repo. And from there they test the examples. I don't expect them to download a single folder. I will update the README anyways